test/suite/test_import01.py

```diff
@@ -64,9 +72,9 @@ def check_record(self, uri, key, value, ts):
     def check(self, uri, keys, values, ts):
```

Comment:
Since the timestamps have to be behind oldest in order to import, it doesn't make sense to read from a range of timestamps so I use the same timestamp for all reads.

We could just do away with the timestamp usage in `check` and just read without one.

Reply:
We should remove the timestamp variable from the check functions if you are sure there's no reason for it.

Reply:
4960bf6